Jenkinsfile

```diff
@@ -662,10 +669,14 @@ pipeline {
                         }
                     }
                 }
+                /* This stage is disabled until we can use local artifactory to
+                install the go package. */
```

Comment:
Do we have a ticket to track turning this back on?

Reply:
There seems to be some confusion about what is going on here.
We build the required version of go locally, but someone at some time blocked it being installed by the installed for leap 15.

Reply:
Turned back on in https://github.com/daos-stack/daos/pull/16224/commits/0172aa3193ecd77fa6c2788a2a1bf92820cb5138 after merging with master to pick up https://github.com/daos-stack/daos/commit/dacee453500a1216d3e934ce0965e2352612254f